### ISSUE TITLE
DoReFa-Net: Training Low Bitwidth Convolutional Neural Networks with Low Bitwidth Gradients

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#61]
# DoReFa-Net: Training Low Bitwidth Convolutional Neural Networks with Low Bitwidth Gradients #

- Author: Shuchang Zhou, Yuxin Wu, Zekun Ni, Xinyu Zhou, He Wen, Yuheng Zou
- Origin: https://arxiv.org/abs/1606.06160
- Related:
  - https://github.com/ppwwyyxx/tensorpack/tree/master/examples/DoReFa-Net
